app/views/comments/reference_comments.js.erb

```diff
@@ -0,0 +1,6 @@
+$("#comments-<%= reference.id %>").html("<%= escape_javascript(render 'papers/comments', reference: reference) %>");
+$("li#comments-tab-<%= reference.id %>").html("<%= escape_javascript(render 'papers/comments_tab', reference: reference) %>");
+$('.toggle-reply').on('click', function(e) {
+  e.preventDefault();
+  $(e.target).parents("ul").next('.comment-reply').toggleClass('hidden');
```

Comment:
We should probably just give each comment reply its own id and select based on that somehow. We can do it later, though.